File: dart/src/n03_animal_race/horse.dart
import 'animal.dart';

class Horse extends Animal {
  Horse() : super('Horse', 2, 80, 100);

  @override
  void makeNoise() {
    print('neigh');
  }

  void throwRider() {
    print('aaaaaaaaaaaaaaaaaaaaaaaa');
  }
}

File: java/src/n03_animal_race/Horse.java
package n03_animal_race;

public class Horse extends Animal {
    public Horse() {
        super("Horse", 2, 80, 100);
    }

    @Override
    public void makeNoise() {
        System.out.println("neigh");
    }

    public void throwRider() {
        System.out.println("aaaaaaaaaaaaaaaaaaaaaaaa");
    }
}
